terraform/modules/kubernetes/main.tf

```diff
@@ -57,12 +57,12 @@ resource "kubernetes_deployment" "webapp" {
           }
           resources {
             requests = {
-              cpu    = "100m"
-              memory = "256Mi"
+              cpu    = var.webapp_cpu_min
+              memory = var.webapp_memory_min
```

Comment:
This should be equal to webapp_memory_max to avoid potential out of memory exceptions

Reply:
done